sist02.csl

```diff
@@ -8,199 +8,133 @@
     <author>
       <name>Shoji Takahashi</name>
       <email>s.takahashi@elsevier.com</email>
-      <uri>http://www.mendeley.com/profiles/shoji-takahashi/</uri>
+      <uri>http://www.mendeley.com/profiles/shoji-takahashi3/</uri>
     </author>
     <contributor>
       <name>Eriko Amano</name>
       <uri>http://www.mendeley.com/profiles/eriko-amano/</uri>
     </contributor>
     <category citation-format="numeric"/>
-    <updated>2014-11-26T05:09:41+00:00</updated>
+    <updated>2015-12-13T06:04:50+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
   <locale xml:lang="ja">
     <terms>
-      <term name="chapter" form="short">ch.</term>
-      <term name="presented at">presented at the</term>
-      <term name="available at">available</term>
+      <term name="open-quote">“</term>
+      <term name="close-quote">”</term>
     </terms>
   </locale>
   <macro name="author">
     <names variable="author" suffix=". ">
       <name initialize="false" initialize-with=". " name-as-sort-order="all"/>
-      <label form="short" prefix=", " text-case="capitalize-first"/>
-      <substitute>
-        <names variable="translator"/>
-      </substitute>
     </names>
   </macro>
-  <macro name="edition">
-    <number prefix="第" variable="edition"/>
-    <text term="edition" form="short" suffix=", "/>
-  </macro>
   <macro name="event">
-    <text variable="event-place" suffix=", "/>
-    <date delimiter="" variable="event-date" suffix=". ">
-      <date-part name="year" range-delimiter="" suffix="-"/>
-      <date-part name="month" form="numeric-leading-zeros" range-delimiter="" suffix="-"/>
-      <date-part name="day" form="numeric-leading-zeros" range-delimiter=""/>
-    </date>
+    <group delimiter=", ">
+      <text variable="event-place"/>
+      <date variable="event-date">
+        <date-part name="year" suffix="-"/>
+        <date-part name="month" form="numeric-leading-zeros" suffix="-"/>
+        <date-part name="day" form="numeric-leading-zeros"/>
+      </date>
+    </group>
   </macro>
   <macro name="title">
     <choose>
-      <if type="article-journal" match="any">
-        <text variable="collection-title" suffix=": "/>
-        <text variable="title" suffix=". "/>
-        <text variable="container-title"/>
-      </if>
-      <else-if type="chapter paper-conference report" match="any">
+      <if type="chapter report paper-conference article webpage" match="any">
         <choose>
-          <if match="any" variable="container-title">
-            <text variable="title" prefix="&quot;" suffix="&quot;. "/>
-            <text variable="container-title" suffix=". "/>
+          <if match="any" variable="container-title" type="webpage">
+            <text term="open-quote"/>
+            <text variable="title" quotes="false"/>
```

Comment:
that's right, I'm no fan of the hardcoded quotes at all, so this is much
better (it will also give you proper typographical curly quotes instead of
ugly straight ones).

On Sun, Dec 13, 2015 at 4:44 PM, Shoji Takahashi <notifications@github.com>
wrote:

> In sist02.csl
> <https://github.com/citation-style-language/styles/pull/1828#discussion_r47451753>
> :
>
> >          <choose>
> > -          <if match="any" variable="container-title">
> > -            <text variable="title" prefix="&quot;" suffix="&quot;. "/>
> > -            <text variable="container-title" suffix=". "/>
> > +          <if match="any" variable="container-title" type="webpage">
> > +            <text term="open-quote"/>
> > +            <text variable="title" quotes="false"/>
>
> Thanks. I just submitted the change. I looked at what I was doing in my
> old style and below is what I did there. I believe you (and I too) hate
> this! <text variable="title" prefix="&quot;" suffix="&quot;. "/> Shoji
> From: Sebastian Karcher [mailto:notifications@github.com] Sent: Monday,
> December 14, 2015 7:34 AM To: citation-style-language/styles Cc: Takahashi,
> Shoji (ELS-TOK) Subject: Re: [styles] Update sist02.csl (#1828
> <https://github.com/citation-style-language/styles/pull/1828>) In
> sist02.csl<#1828 (comment)
> <https://github.com/citation-style-language/styles/pull/1828#discussion_r47451593>
> >:
> <choose> - <if match="any" variable="container-title"> - <text
> variable="title" prefix="&quot;" suffix="&quot;. "/> - <text
> variable="container-title" suffix=". "/> + <if match="any"
> variable="container-title" type="webpage"> + <text term="open-quote"/> +
> <text variable="title" quotes="false"/>
> I'd much prefer that, yes. I've double-checked 2.) in Zotero and it works
> correctly there, too. One possibility is that Mendeley is simply using an
> older version of citerproc-js, in which case this will fix itself, but
> would be good to check with them, yes. I'd much rather not have the
> workaround in the style, though. — Reply to this email directly or view it
> on GitHub<
> https://github.com/citation-style-language/styles/pull/1828/files#r47451593
> >.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/citation-style-language/styles/pull/1828/files#r47451753>
> .
>



-- 
Sebastian Karcher, PhD
www.sebastiankarcher.com